build.gradle.kts

```diff
@@ -42,7 +42,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath(libs.sqldelight.plugin)
+//        classpath(libs.sqldelight.plugin)
```

Comment:
Shall we delete this?

Reply:
[clean comment](https://github.com/Mindera/Android-KMP-Template/pull/9/commits/e09fb404c1d6269834678a8fc8f1a294cb036c3c)